code/script.js

```diff
@@ -261,75 +271,151 @@ const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
   // This variable stores what option group (category) the question belongs to.
   // We also need a variable that stores the actual value of the question we've selected.
+  //console.log(category) //hair color, eye color, accessories, other
+   const value = questions.options[questions.selectedIndex].value
+  //console.log(value)// brown, blue, glasses, smoker
 
   if (category === 'hair color') {
     currentQuestion = {
       attribute: 'hairColor',
-      // value: ,
+      value: value,
```

Comment:
If you dont feel like writing value:value or category:category i think you can just write value: and category: and leave it empty, but only if its the same word.

Reply:
And this will save you tons of time in the bigger projects, and will it readable if you want to write a short object in just one line 🤓

Saved time can be used to rewatch "My Little Pony"🦄  or whatever is your poison of choice 🎥